pkg/api/server/userstored/proxy_store.go

```diff
@@ -25,10 +25,10 @@ func addProxyStore(ctx context.Context, schemas *types.Schemas, context *config.
 
 	prefix := []string{"api"}
 	group := ""
-	version := "v1"
 	kind := s.CodeName
 	plural := s.PluralName
 
+	var version string
```

Comment:
nit: it might look better to bind it with the variable above: `var group, version string`

Reply:
updated